gradle-plugins/compose/src/test/test-projects/application/jvm/settings.gradle

```diff
@@ -0,0 +1,11 @@
+pluginManagement {
+    plugins {
+        id 'org.jetbrains.kotlin.jvm' version '1.4.10'
```

Comment:
Interesting, why 1.4.10? Maybe 1.4.20 (latest) or 1.4.0 (used in Compose)?

Reply:
Fixed